packages/prisma/src/prisma/schema.prisma

```diff
@@ -959,10 +958,6 @@ model LeaderboardEntry {
   sessionParticipation   Participation? @relation("SessionLeaderboards", fields: [sessionParticipationId], references: [id])
```

Comment:
should we also remove SESSION_BLOCK from the LeaderboardType enum if we are not going to use it?

Reply:
absolutely, good idea

Reply:
has been fixed